### PR TITLE
Run rrdcached service as rrdcached_t

### PR DESCRIPTION
**Pull request by 5umm3r15**
Fixed Red Hat Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1726255#

### CODE REVIEW

File: rrdcached.fc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/usr/bin/rrdcached		--	gen_context(system_u:object_r:rrdcached_exec_t,s0)
+
+/var/run/rrdcached.pid		--	gen_context(system_u:object_r:rrdcached_var_run_t,s0)
```
Comment (ghost) [inline rrdcached.fc]: Probably best to escape the period as regular expressions interpret period as anything:

`/var/run/rrdcached\.pid`

File: rrdcached.if [hunk 2]
```diff
@@ -0,0 +1,97 @@
+
+## <summary>policy for rrdcached</summary>
```
Comment (ghost) [inline rrdcached.if]: Daemon that receives updates to existing RRD files, accumulates them and writes the updates to the RRD file.

File: rrdcached.te [hunk 3]
```diff
@@ -0,0 +1,45 @@
+policy_module(rrdcached, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type rrdcached_t;
+type rrdcached_exec_t;
+type tmp_t;
```
Comment (ghost) [inline rrdcached.te]: Duplicate declaration and you are not allowed to reference external identifiers directly:

```
type rrdcached_tmp_t;
files_tmp_file(rrdcached_tmp_t)
```

File: rrdcached.te [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+#
+allow rrdcached_t self:capability { chown setgid setuid };
+allow rrdcached_t self:process { fork signal_perms };
+allow rrdcached_t self:fifo_file rw_fifo_file_perms;
+allow rrdcached_t self:unix_stream_socket create_stream_socket_perms;
+
+allow rrdcached_t tmp_t:dir { add_name write };
+allow rrdcached_t tmp_t:dir remove_name;
+allow rrdcached_t tmp_t:sock_file create;
+allow rrdcached_t tmp_t:sock_file unlink;
```
Comment (ghost) [inline rrdcached.te]: not allowed to reference external identifiers directly:
```
allow rrdcached_t rrdcached_tmp_t:sock_file manage_sock_file_perms;
files_tmp_filetrans(rrdcached_t, rrdcached_tmp_t, sock_file)
```

File: rrdcached.te [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+files_read_etc_files(rrdcached_t)
+
+auth_use_nsswitch(rrdcached_t)
+
+logging_send_syslog_msg(rrdcached_t)
+
+miscfiles_read_localization(rrdcached_t)
+
+sysnet_dns_name_resolve(rrdcached_t)
```
Comment (ghost) [inline rrdcached.te]: Redundant as this is already implied by `auth_use_nsswjtch(rrdcached_t)`

**Review by ghost — COMMENTED** [review]
Some observations

**Comment by wrabcak** [comment]
LGTM. 
Thanks!

**Pull request merged**